### PR TITLE
Add release note for 1.1 release

### PR DESCRIPTION
**Pull request by VijayanB**
Signed-off-by: Vijayan Balasubramanian <balasvij@amazon.com>

### Description
Add release note for 1.1 release
 
 
### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass
- [ ] New functionality has been documented.
  - [ ] New functionality has documentation added
- [x] Commits are signed as per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/opensearch-cli/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by vamshin — DISMISSED** [review]
LGTM! Thanks

File: release-notes/opensearch-cli.release-notes.1.1.0.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+## Version 1.1.0 Release Notes
+
+### Enhancements
+
+* Fix config file permission ([#29](https://github.com/opensearch-project/opensearch-cli/pull/29))
+* Update version to next minor version. ([#30](https://github.com/opensearch-project/opensearch-cli/pull/30))
```
[the remark this answers is not in the document]
Comment (VijayanB) [reply release-notes/opensearch-cli.release-notes.1.1.0.md]: Ack.

File: release-notes/opensearch-cli.release-notes.1.1.0.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+## Version 1.1.0 Release Notes
+
+### Enhancements
+
+* Fix config file permission ([#29](https://github.com/opensearch-project/opensearch-cli/pull/29))
```
Comment (jmazanec15) [inline release-notes/opensearch-cli.release-notes.1.1.0.md:5]: Is this a bug fix or an enhancement?

Comment (VijayanB) [reply release-notes/opensearch-cli.release-notes.1.1.0.md]: Updated title to reflect this as enhancement.

**Review by jmazanec15 — APPROVED** [review]

**Review by naveentatikonda — APPROVED** [review]

**Pull request merged**